rustls-libssl/src/lib.rs

```diff
@@ -1259,6 +1328,40 @@ impl From<VerifyMode> for i32 {
     }
 }
 
+#[derive(Debug, Default, Clone)]
+struct EnabledVersions {
+    min: Option<ProtocolVersion>,
+    max: Option<ProtocolVersion>,
+}
+
+impl EnabledVersions {
+    fn reduce_versions(
+        &self,
+        method_versions: &'static [&'static rustls::SupportedProtocolVersion],
```

Comment:
nit: since `SupportedProtocolVersion` is used in a couple places, maybe worth an import to drop the `rustls::` prefix?